pages/linux/importctl.md

```diff
@@ -0,0 +1,20 @@
+# importctl
+
+> Download, Import, or Export disk images.
+> For More Information: <https://man7.org/linux/man-pages/man1/importctl.1.html>.
+
+- [d]ownload image in tarball format from a url via pull:
+
+`sudo importctl pull-tar {URL} {path/to/directory}`
+
+- [p]ull or download from a remote source that is either raw or qcow2 file, and stores it as a raw file:
+
+`sudo importctl pull-raw {{https:\\url\to\source.ext}} {{name}} --class={{[machine|portable|sysext|confext]}}`
```

Comment:
```suggestion
`sudo importctl pull-raw {{https:\\url\to\source.ext}} {{name}} --class={{machine|portable|sysext|confext}}`
```
This is not correct. Read our documentation for what an option placeholder is
https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#page-structure